src/actions/serve.js

```diff
@@ -35,5 +35,14 @@ module.exports = () => {
   subprocess.catch(e => console.log(chalk.red(e.stderr)))
   subprocess.stdout.pipe(process.stdout)
 
+  // Hack for windows to prevent leaving orphan processes, resulting in multiple http-server running instances
+  if (require('os').platform() === 'win32') {
+    process.on('SIGINT', () => {
+      require('child_process').exec('taskkill /PID ' + subprocess.pid + ' /T /F', () => {
```

Comment:
same here, better to do something like `const child_process = require('child_process')` in the top of the file.

furthermore a question: the arguments passed to `taskkill` are uppercase, whereas the documentation shows them lowercase (https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill) .. any reason?